components/mpas-ocean/docs/user-guide/index.md

```diff
@@ -0,0 +1,210 @@
+
+`GMPAS-JRA1p5-DIB-PISMF` - Active ocean-sea ice configuration forced by JRA55 v1.5 atmosphere (as above), with data iceberg and data ice-shelf melt
+
+`CMPASO-JRA1p4` - Active ocean configuration forced by data atmosphere based on JRA55 v1.4 (covers 61 years, 1958-2018)
+
+`CMPASO-IAF` - Active ocean configuration forced by data atmosphere based on CORE-II (covers 62 years, 1948-2009)
+
+Additional compsets can be found in the [mpas-ocean `config_compsets.xml`](https://github.com/E3SM-Project/E3SM/blob/master/components/mpas-ocean/cime_config/config_compsets.xml). Note that the fully coupled compsets and their aliases can be found in the [cime allactive `config_compsets.xml`](https://github.com/E3SM-Project/E3SM/blob/master/cime_config/allactive/config_compsets.xml).
+For more information on the schemes used within MPAS-Ocean, refer to the [MPAS-Ocean Users Guide](../tech-guide/mpas_ocean_users_guide.pdf).
+
+A full list of Compsets in the current repository can be listed using
+
+```text
+cd cime/scripts
+./query_config --compsets
+```
+
+### Meshes
+
+Supported meshes for G- and C-cases include:
+
+`TL319_IcoswISC30E3r5` - Icosahedral 30 km mesh with ice shelves cavities (wISC), E3SMv3 (E3) revision r5, TL319 is the grid for JRA.
+
+`T62_IcoswISC30E3r5` - Icosahedral 30 km mesh with ice shelves cavities (wISC), E3SMv3 (E3) revision r5, T62 is the grid for CORE-II.
+
+`T62_oQU240` - Quasi uniform 2-degree ocean mesh, to be used with CORE-II only. Good for rapid testing, used in nightly testing **not** production runs.
+
+Note: the mesh should be consistent with the compset (e.g. JRA vs CORE). Additional mesh information can be found [here](https://github.com/E3SM-Project/E3SM/blob/master/cime_config/config_grids.xml).
+
+A full list of Meshes in the current repository can be listed using
+
+```text
+cd cime/scripts
+./query_config --grids
+```
+
+## Customizing runs
+
+### Namelist changes
+
+Without additional input, E3SM will generate the namelist file `mpaso_in` in the run directory using the default values for the compset and mesh requested.
+Namelist parameters can be changed from default values by modifying the `user_nl_mpaso` file, found in the `case_scripts` directory. This is done by entering ``[namelist option] = [changed value]`` as separate lines in the ``user_nl_mpaso`` file. All other options will remain defaults. These changes can be added at run time and will take effect in the next submission.
+
+Refer to the [MPAS-Ocean Users Guide](../tech-guide/mpas_ocean_users_guide.pdf) (Chapter 10) for a comprehensive description of the namelist parameters and the options that they correspond to. Namelist options may also be found in the code repository in the file `components/mpas-ocean/src/Registry.xml` for general flags and `components/mpas-ocean/src/tracer_groups/Registry_*.xml` for specific tracer group flags.
```

Comment:
may not be important but analysis member flags are in yet a different spot `components/mpas-ocean/src/analysis_members/Registry_*.xml`